tests/jest/config/webpack/loaders/devSiteCssFinder.test.jsx

```diff
@@ -0,0 +1,33 @@
+import findCss from '../../../../../config/webpack/loaders/devSiteCssFinder';
+
+describe('devSiteExample css finder', () => {
+  it('separates a shortened css file path correctly', () => {
+    const testText = 'Import from \'./pretendExampleCss.scss\'';
+    const example = findCss(testText);
+    expect(example).toMatchSnapshot();
+  });
+
+  it('separates a direct css file correctly', () => {
+    const testText = 'Import from \'/Users/dm068655/Documents/ExternalR/terra-dev-site/tests/jest/config/webpack/loaders/pretendExampleCss.scss\'';
+    const example = findCss(testText);
+    expect(example).toMatchSnapshot();
+  });
+
+  it('returns undefined if there is no .css or .scss file declaration', () => {
```

Comment:
Can we add some tests for css.moudle.scss?

Also can we add a few tests where multiple css imports are passes in to ensure the first path is resolved?

Reply:
I added a test to make sure it picked up the whole file name for css.module.scss and a second to ensure that the first import path is the one that is loaded [d830ef0](https://github.com/cerner/terra-dev-site/pull/281/commits/d830ef07a820fb004bc59f65cde8f20ee7052353#).
I can add more for css.module.scss if you had something different in mind?